source/swap/test/Swap.js

```diff
@@ -1,14 +1,14 @@
 const Swap = artifacts.require('Swap')
 const Types = artifacts.require('Types')
 const FungibleToken = artifacts.require('FungibleToken')
-const NonFungibleToken = artifacts.require('NonFungibleToken')
-const OMGToken = artifacts.require('OMGToken')
-
+const TransferHandlerRegistry = artifacts.require('TransferHandlerRegistry')
```

Comment:
Why have you removed all ERC721 tests from this file? I do see that this is now a test of swap not so much the transfer handlers, but I can't see any tests that test the different transfer handlers either. I think it's important we leave the ERC721 tests in here (or add a file that tests the ERC721 transfer handler).

Reply:
These tests have been moved to transfers/TransferHandler.js